src/i2c.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <stdint.h>
+#include <libopencm3/stm32/i2c.h>
+#include <libopencm3/stm32/rcc.h>
+#include <libopencm3/stm32/gpio.h>
+
+#define I_SENSE_LSB 0.001
```

Comment:
question: Where has this value come from? Datasheet refs?

Reply:
This is 1mA which is the resolution specified in the interface document